### ISSUE TITLE
(demo): FlexLayout Splitter demo not working on IE11

### ISSUE THREAD

**Issue by ankur021188** [angular/flex-layout#526]
FlexBox Splitter not working on IE 11. 

![use](https://content.screencast.com/users/amcdaniel22/folders/Snagit/media/636877b3-a722-47a8-a17a-3ab40fd0ecba/2017-04-25_10-02-30.png)

#### Usage
```javascript
<div fxLayout="row" ngxSplit="row">
  <div fxFlex="30%" ngxSplitArea>
    Left
  </div>
  <div fxFlex="5px">
    <a href="#" ngxSplitHandle>...</a>
  </div>
  <div fxFlex="70%" ngxSplitArea>
    Right
  </div>
</div>
```

> See https://github.com/angular/flex-layout/issues/266 is not working.

Please provide the steps to resolve the issue.

The main reason for this issue is when we start dragging, after a moment when `SplitAreaDirective.activatedValue` starts returning values which are actually decimal numbers, in IE the value returned is `NaN`.

So, how to resolve the issue?

**Comment by CaerusKaru** [comment]
Ok, I _finally_ have the root cause analysis on this. It's not because of `activatedValue`, it's because IE doesn't support `MouseEvent.movementX` and `MouseEvent.movementY`, which we use to track drag events. We just need to find another API that is supported in all browsers and switch to that.

**Comment by CaerusKaru** [comment]
This will finally, _finally_ be patched by #922.
